### ISSUE TITLE
Add Chocolatey package updates in build pipeline or Actions

### ISSUE THREAD

**Issue by Vacant0mens** [rabbitmq/rabbitmq-server#8866]
### Is your feature request related to a problem? Please describe.

The listing for the [rabbitmq chocolatey pacakge](https://community.chocolatey.org/packages/rabbitmq#versionhistory) seems like it has gotten updates about once or twice per month. The latest package in there (for `v3.11.17`) was updated a day after the respective release on GitHub. The releases updated on Chocolatey.org seem to have followed closely to the releases on GitHub until the end of May this year.

Specifically, the rabbitmq/chocolatey-package repository lists `3.12.x` as one if its branches, but there have been no `3.12.x` releases to chocolatey yet. I also see that the `3.12` branch has been updated from `3.12.0` to `3.12.1` recently (committed by `rabbitmq-ci`), but neither of those have shown up on chocolatey yet. However, I don't see a `3.11` branch in the list for that repo.

### Describe the solution you'd like

If it's not already there/available, is it possible to integrate the [chocolatey-package](https://github.com/rabbitmq/chocolatey-package) repo into the build process so that the releases to Chocolatey are automatic alongside the GitHub releases?

Clearly there are (and/or were) already some elements of CI/automation to update the `rabbitmq/chocolatey-package` repository. Has support for Chocolatey been put on hold for some reason?

**Comment by lukebakken** [comment]
Yes, we have an internal Concourse pipeline to publish to Chocolatey. Here is the current status:

![image](https://github.com/rabbitmq/rabbitmq-server/assets/514926/2d69fd9b-7985-428f-a096-c2e7604f7cb1)

When we have time we'll investigate why the pipelines aren't working correctly. Thanks for bringing this up.
